plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSink.java

```diff
@@ -409,7 +413,7 @@ private int[] getWriterIndexes(Page page)
                 writers.set(writerIndex, writer);
                 memoryUsage += writer.getMemoryUsage();
             }
-            catch (IOException e) {
+            catch (RuntimeException e) {
```

Comment:
Can we still use `IOException` here?

Reply:
No we can't it is no longer thrown here. Previously it was thrown by `hdfsEnvironment.getFileSystem(session.getIdentity(), rootTableLocation, conf)`